tests/sentry/api/serializers/test_alert_rule.py

```diff
@@ -87,3 +94,203 @@ def test_triggers(self):
         result = serialize([alert_rule, other_alert_rule], serializer=DetailedAlertRuleSerializer())
         assert result[0]["triggers"] == [serialize(trigger)]
         assert result[1]["triggers"] == []
+
+
+class CombinedRuleSerializerTest(BaseAlertRuleSerializerTest, APITestCase, TestCase):
```

Comment:
`ProjectCombinedRuleIndexEndpointTest`, and this belongs in tests/sentry/incidents/endpoints/test_project_alert_rule_index.py